cgsmiles/resolve.py

```diff
@@ -177,9 +177,7 @@ def edges_from_bonding_descrpt(self):
 
             # bonding descriptors are assumed to have bonding order 1
             # unless they are specifically annotated
-            order = re.findall("\d+\.\d+", bonding[0])
-            if not order:
-                order = 1
+            order = int(bonding[0][-1])
```

Comment:
Breaks if you're insane and want to add an aromatic bond

Reply:
Actually no it doesn't; here your bond order will implicitly become 1 and afterwards the kekulization in pysmiles takes care of giving you the correct bond orders IF you build the correct aromatic molecule otherwise you get the more correct bond order anyways.